azure-pipelines.yml

```diff
@@ -230,6 +230,7 @@ stages:
               Token: $(dn-bot-dnceng-artifact-feeds-rw)
         - script: eng\common\cibuild.cmd
             -configuration $(_BuildConfig)
+            -msbuildEngine dotnet
```

Comment:
Hmm what is this trying to fix? Last I worked on this PR I had errors on the windows build with 
```
##[error].dotnet\sdk\5.0.100-preview.5.20258.4\Sdks\Microsoft.NET.Sdk\targets\Microsoft.PackageDependencyResolution.targets(234,5): error NETSDK1005: (NETCORE_ENGINEERING_TELEMETRY=Build) Assets file 'F:\workspace\_work\1\s\artifacts\obj\RazorPageGenerator\project.assets.json' doesn't have a target for '.NETCoreApp,Version=v5.0'. Ensure that restore has run and that you have included 'net5.0' in the TargetFrameworks for your project.
```

But now I'm seeing errors on every platform:
```
##[error].dotnet\sdk\5.0.100-preview.5.20258.4\Sdks\Microsoft.NET.Sdk\targets\Microsoft.PackageDependencyResolution.targets(234,5): error NETSDK1005: (NETCORE_ENGINEERING_TELEMETRY=Build) Assets file 'F:\workspace\_work\1\s\artifacts\obj\RazorPageGenerator\project.assets.json' doesn't have a target for '.NETCoreApp,Version=v5.0'. Ensure that restore has run and that you have included 'net5.0' in the TargetFrameworks for your project.
```

Not sure if this is the cause though.

Reply:
Actually the windows runs are failing with that error but the mac/linux are failing tests. Looking.

Reply:
Ah I see why this change is made now.